packages/core/content-type-builder/server/services/schema-builder/content-type-builder.js

```diff
@@ -100,6 +101,10 @@ module.exports = function createComponentBuilder() {
         .set('kind', infos.kind || typeKinds.COLLECTION_TYPE)
         .set('collectionName', infos.collectionName || defaultCollectionName)
         .set('info', {
+          singularName: modelName,
+          pluralName: pluralize(modelName),
+          displayName: infos.name,
+          // TODO: remove name eventually
```

Comment:
Why note removing it directly?

Reply:
We still use in in the CM & CTB. It doesn't make sense to do this in that PR I think :)